styles/modules.css

```diff
@@ -46,7 +47,7 @@
 /*MOBILE FIRST @MEDIA============*/
 
 @media only screen and (max-width: 960px) {
```

Comment:
**TODO FOR LOST POINTS**

This is only a theory for now, but your media queries on the hamburger menu don't work. I think it's because more specific CSS takes precedence on an element, and the CSS link you're using for the hamburger menu:

```
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

is taking precedence over your own CSS. Can you try using the vendor CSS files instead similar to starter code?
https://github.com/codefellows-seattle-301d18/03-jQuery-and-events/tree/master/starter-code/vendor

And replace hamburger menu with the hamburger menu from the ``fonts.css`` folder. I think it's the attribute ``.icon-menu:before {
	content: "\e902";
}``.